apps/extension/webpack.config.ts

```diff
@@ -0,0 +1,153 @@
+import * as dotenv from 'dotenv';
+import path from 'path';
+import CopyPlugin from 'copy-webpack-plugin';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import url from 'url';
+// eslint-disable-next-line import/no-named-as-default
+import webpack from 'webpack';
```

Comment:
sadly `import * as webpack from 'webpack';` does not work for this like most cases